draft-integration.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: integration
+name: Name of your integration
+description: Description of integration
+authors:
+    - name: author-name
+      socials:
+        github: author-username
+        twitter: author-username
+        linkedin: author-username
+pypi: https://pypi.org/project/{project-slug}
+repo: https://your-repo-url
+type: Type of integration
```

Comment:
Let's also give them some options to start with: "Model Provider", "Document Store"